.github/workflows/release.yaml

```diff
@@ -66,28 +66,28 @@ jobs:
           git commit -m "Release ${{steps.prepare-release.outputs.next-release-tag}}"
           git push origin HEAD:main
 
-      - name: Install dependencies
-        run: flutter pub get
+      # - name: Install dependencies
```

Comment:
should just delete these comments if not going to use it

Reply:
this allows us a quick revert - will make a separate pr to remove if it works